sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -58,15 +58,8 @@ class AudioAndCCSelectionPanel extends Component {
   }
 
   onAudioTrackSelected = (name) => {
-    const localizedTitleForUndefinedLanguage = Utils.localizedString(this.props.config.locale, stringConstants.undefinedLanguageTitle, this.props.config.localizableStrings);
-    const localizedTitleForNoLinguisticContent = Utils.localizedString(this.props.config.locale, stringConstants.noLinguisticContentTitle, this.props.config.localizableStrings);
-    let originalName = name;
-
-    originalName = originalName.replace(localizedTitleForUndefinedLanguage, stringConstants.undefinedLanguageTitle);
-    originalName = originalName.replace(localizedTitleForNoLinguisticContent, stringConstants.noLinguisticContentTitle);
-
-    if (this.props.selectedAudioTrackTitle !== originalName) {
-      this.props.onSelectAudioTrack(originalName);
+    if (this.props.selectedAudioTrackTitle !== name) {
```

Comment:
Was it an intentional change?

Reply:
No, probable it was inherited from `dev`-branch